app/views/reports/standard_form_report/subset/_tags.html.erb

```diff
@@ -0,0 +1,10 @@
+<% # print tag header if grouped by tag %>
```

Comment:
Maybe call this file `tag_header` to be a bit more specific?

Reply:
done